framework/CHANGELOG.md

```diff
@@ -4,45 +4,46 @@ Yii Framework 2 Change Log
 2.0.12 under development
 --------------------------
 
-- Bug #13671: Fixed error handler trace to work correctly with XDebug (samdark)
```

Comment:
As far as I remember, when I did the same thing (reordering of the whole CHANGELOG's section in one of my PRs) the other core members reacted negatively and explained that this is not the right thing to do because the diff becomes confusing. Should be done out of scope of this PR. Hopefully we'll fix #13482 and this will not be a problem anymore.